src/lib/Components/Lists/SavedShowItemRow.tsx

```diff
@@ -86,7 +86,7 @@ export class SavedShowItemRow extends React.Component<Props, State> {
               },
             },
             updater: store => {
-              store.get(relayID).setValue(!isShowFollowed, "is_followed")
+              store.get(nodeID).setValue(!isShowFollowed, "is_followed")
```

Comment:
oh nice! so we this `store...setValue` we don't need to refetch everything after save?

Reply:
Yeah there is an imperative API to the Relay store.